### PR TITLE
WIP: Changes in libxml2.cmake

### PR DESCRIPTION
**Pull request by e2l-su**
This is second release of libxml2.cmake which adds support of libxml2 within Dynaωo.

### CODE REVIEW

File: dynawo/3rdParty/libxml2/libxml2.cmake [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+
+cmake_minimum_required(VERSION 3.12)
+
+set(packet_name        "libxml2")
```
Comment (gautierbureau) [inline dynawo/3rdParty/libxml2/libxml2.cmake:13]: package is way better

**Review by gautierbureau — APPROVED** [review]

File: dynawo/3rdParty/libiidm/libiidm.cmake [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+# file, you can obtain one at http://mozilla.org/MPL/2.0/.
+# SPDX-License-Identifier: MPL-2.0
+#
+# This file is part of Dynawo, an hybrid C++/Modelica open source time domain simulation tool for power systems.
+
+cmake_minimum_required(VERSION 3.12)   
+
+include($ENV{DYNAWO_HOME}/dynawo/cmake/CPUCount.cmake)
+if(NOT DEFINED CPU_COUNT ) 
+  message(FATAL_ERROR "Le décompte des CPUs a échoué")
```
Comment (rosiereflo) [inline dynawo/3rdParty/libiidm/libiidm.cmake:15]: No french.

**Review by rosiereflo — CHANGES_REQUESTED** [review]

**Comment by e2l-su** [comment]
God damned, nobody is perfect !


On Mon, 21 Sep 2020 04:16:39 -0700
rosiereflo <notifications@github.com> wrote:

> @rosiereflo requested changes on this pull request.
> 
> 
> 
> > @@ -0,0 +1,79 @@
> +# Copyright (c) 2015-2020, RTE (http://www.rte-france.com)
> +# See AUTHORS.txt
> +# All rights reserved.
> +# This Source Code Form is subject to the terms of the Mozilla Public
> +# License, v. 2.0. If a copy of the MPL was not distributed with this
> +# file, you can obtain one at http://mozilla.org/MPL/2.0/.
> +# SPDX-License-Identifier: MPL-2.0
> +#
> +# This file is part of Dynawo, an hybrid C++/Modelica open source
> time domain simulation tool for power systems. +
> +cmake_minimum_required(VERSION 3.12)   
> +
> +include($ENV{DYNAWO_HOME}/dynawo/cmake/CPUCount.cmake)
> +if(NOT DEFINED CPU_COUNT ) 
> +  message(FATAL_ERROR "Le décompte des CPUs a échoué")
> 
> No french.
> 



-------------------------

Cordialement, Denis Gillain, http://www.ellipsys.fr
__________________________________________________

E      L      L      I      P      S      Y      S

+33 (0) 1 42 36 37 19 -  denis.gillain@ellipsys.fr
__________________________________________________

File: dynawo/3rdParty/libiidm/libiidm.cmake (hunk 2 above)
Comment (e2l-su) [reply dynawo/3rdParty/libiidm/libiidm.cmake:15]: That little part is included in all others. It's a common part which can be mutualized. That code already exists in a few CMakeLists.txt but as copy/paste. I only take it apart to share it (which means testing is done good).

Comment (e2l-su) [reply dynawo/3rdParty/libiidm/libiidm.cmake:15]: done, nobody is perfect

File: dynawo/3rdParty/libxml2/libxml2.cmake (hunk 1 above)
Comment (e2l-su) [reply dynawo/3rdParty/libxml2/libxml2.cmake]: that's right. Corrected

**Comment by e2l-su** [comment]
I close that PR since changes are done, and new PR follows

**Pull request closed without merging**